### DISCUSSION TITLE
Permitir que las carpetas se compartan en las organizaciones

### DISCUSSION THREAD

**Discussion by WesTliFe212** [dani-garcia/vaultwarden#2839] [Ideas]
Buenas Dani. Me gustaría saber si puedes añadir algo para que podamos compartir las carpetas de las organizaciones. Gracias.

**Comment by BlackDex** [comment]
Translation:
> Good Dani. I would like to know if you can add something so that we can share the folders of the organizations. Thanks.

I'm afraid that isn't possible. This kind of functionality needs to be provided by Bitwarden it self via the clients.
Since Vaultwarden doesn't make any changes to the clients that is not something Vaultwarden can provide.

I suggest to request a feature like this, if there isn't already, at Bitwarden.

A work-around would be using collections and put everything in separate collections.

**Reply by WesTliFe212** [reply]
Es posible visualizar las colecciones cuando realizamos una búsqueda en vaultwarden?

**Reply by BlackDex** [reply]
I Would suggest to check this page from Bitwarden: https://bitwarden.com/help/searching-vault/

You can't see the specific collections as far as i know.
You can click on a collection and after that search within that collection, but that is about it.
Only other way would be to open that specific cipher and see to which collections it is linked.
